core/src/main/java/org/opensearch/sql/expression/Expression.java

```diff
@@ -3,44 +3,35 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-
 package org.opensearch.sql.expression;
 
 import java.io.Serializable;
 import org.opensearch.sql.data.model.ExprValue;
 import org.opensearch.sql.data.type.ExprType;
 import org.opensearch.sql.expression.env.Environment;
 
-/**
- * The definition of the resolved expression.
- */
+/** The definition of the resolved expression. */
 public interface Expression extends Serializable {
 
-  /**
-   * Evaluate the value of expression that does not depend on value environment.
-   */
+  /** Evaluate the value of expression that does not depend on value environment. */
   default ExprValue valueOf() {
     return valueOf(null);
   }
 
-  /**
-   * Evaluate the value of expression in the value environment.
-   */
+  /** Evaluate the value of expression in the value environment. */
   ExprValue valueOf(Environment<Expression, ExprValue> valueEnv);
 
-  /**
-   * The type of the expression.
-   */
+  /** The type of the expression. */
   ExprType type();
 
   /**
    * Accept a visitor to visit current expression node.
-   * @param visitor   visitor
-   * @param context   context
-   * @param <T>       result type
-   * @param <C>       context type
-   * @return          result accumulated by visitor when visiting
+   *
+   * @param visitor visitor
+   * @param context context
+   * @param <T> result type
+   * @param <C> context type
+   * @return result accumulated by visitor when visiting
```

Comment:
would adding pre tags here help fix the even spacing from being lost?

Reply:
It would, but I don't think it's worth it as @oaram are important parts of Javadocs so I think we shouldn't mess with it.